en/docs/learn/design-api/advanced-topics/control-api-visibility-and-subscription-availability-in-developer-portal.md

```diff
@@ -123,6 +123,48 @@ When using the REST API directly, the subscription availability options are avai
     </tr>
     </tbody>
     </table>
+    
+    
+## Control API Document Visibility in Developer Portal
+
+By default, any document associated with an API has the same visibility level of the API. That is, if the API is public, 
+its documentation is also visible to all users (registered and anonymous).
+
+!!! note
```

Comment:
There should be a section "Enabling document visibility in Developer Portal", and the instructions should come under that.

Reply:
fixed in 9234c7c